example/spec/parallel/testAPI/collectionSpec.js

```diff
@@ -46,13 +52,43 @@ describe('Collections API', () => {
         bucket: config.bucket,
         key: recordCreatedKey,
       })).toBeResolved();
+      const savedEvent = await getJsonS3Object(config.bucket, recordCreatedKey);
+      const message = JSON.parse(savedEvent.Records[0].Sns.Message);
+      expect(message.event).toEqual('Create');
+      expect(omit(message.record, ['createdAt', 'updatedAt'])).toEqual(omit(collection, ['createdAt', 'updatedAt']));
+    }
+  });
+
+  it('updating a collection publishes a record to the collection reporting SNS topic', async () => {
+    if (beforeAllFailed) {
+      fail('beforeAll() failed');
+    } else {
+      const updatedCollection = {
+        ...collection,
+        reportToEms: false,
+      };
+      await updateCollection({
+        prefix: config.stackName,
+        collection: updatedCollection,
+      });
+
+      await expectAsync(waitForObjectToExist({
+        bucket: config.bucket,
+        key: recordUpdatedKey,
+      })).toBeResolved();
+      const savedEvent = await getJsonS3Object(config.bucket, recordUpdatedKey);
+      const message = JSON.parse(savedEvent.Records[0].Sns.Message);
+      expect(message.event).toEqual('Update');
+      expect(omit(message.record, ['createdAt', 'updatedAt', 'files'])).toEqual(omit(updatedCollection, ['createdAt', 'updatedAt', 'files']));
+      expect(message.record.reportToEms).toEqual(false);
```

Comment:
We probably want to check that `reportToEms` is true after creating the record so that we can prove that the record was definitely updated

Reply:
[`8d76b1c` (#2432)](https://github.com/nasa/cumulus/pull/2432/commits/8d76b1c48461cee69ebff33bd04b3d57dc3c0796)

Reply:
We don't need/want to omit properties if they're coming back from the database, because it gives us an even more precise test

Reply:
We are still omitting properties in the comparison, which is unnecessary if we're comparing against the record fetched from the API

Reply:
[`68b2fb6` (#2432)](https://github.com/nasa/cumulus/pull/2432/commits/68b2fb64fb360c103d4a24de28286ad68b4c75fb)

Sorry to have missed this the first time you mentioned it!